ocs_ci/ocs/ui/page_objects/block_and_file.py

```diff
@@ -259,3 +260,102 @@ def get_avg_consumption_from_ui(self):
         )
         logger.info(f"'Average of storage consumption per day' from the UI : {average}")
         return average
+
+    def get_block_file_encryption_summary(self):
+        """
+        Click on Encryption Summary button and retrieve the encryption details.
+
+        Returns:
+            dict: Encryption summary on block and file page.
+        """
+        encryption_summary = {
+            "cluster_wide_encryption": {"status": None, "kms": ""},
+            "storageclass_encryption": {
+                "status": None,
+                "kms": "",
+            },
+            "intransit_encryption": {"status": None},
+        }
+
+        logger.info("Getting Block and File Encryption Summary Details")
+
+        # Open the encryption summary popup
+        self.do_click(
+            self.validation_loc["encryption_summary"]["file_and_block"]["enabled"],
+            enable_screenshot=True,
+        )
+        from time import sleep
```

Comment:
There must be some other way to check it.
You can try this https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/base_ui.py#L490 or other wait functions for elements.